assets/airlines/est.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icao": "est",
+    "name": "Nordica",
```

Comment:
This airline is very confusing. It is now called Xfly apparently!
> In February 2020, Regional Jet announced it is rebranded as Xfly. Furthermore, an additional seven Embraer 190/195 will be leased to expand operations.
(see https://en.wikipedia.org/wiki/Nordica_(airline)

Reply:
I agree that they are super confusing! I have changed their name to Xfly. Furthermore, I really don't think they actually went through with expanding their operations due to the pandemic, so I won't add their new shiny Embraer.